budget/migrations/env.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import with_statement
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+from logging.config import fileConfig
+import logging
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
```

Comment:
nit: please start comments with a capital and end them with a dot. Wrap them at 80 chars (not less).